app/models/ingredient.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+class Ingredient < ApplicationRecord
+  DOES_NOT_MATCH = "doesn't match an ingredient that exists in Skyrim"
+
+  belongs_to :game
+  belongs_to :canonical_ingredient,
+             class_name: 'Canonical::Ingredient',
+             optional: true,
+             inverse_of: :ingredients
+
+  has_many :ingredients_alchemical_properties, dependent: :destroy, inverse_of: :ingredient
+  has_many :alchemical_properties,
+           -> { select 'alchemical_properties.*, ingredients_alchemical_properties.priority' },
```

Comment:
We ultimately might want `strength_modifier` and `duration_modifier` attributes to be exposed as well, but since we're not using that right now in the code, we'll save that for when we need it.